stestr/test_processor.py

```diff
@@ -68,10 +68,14 @@ class TestProcessorFixture(fixtures.Fixture):
         to use for the run
     :param int concurrency: How many processes to use. The default (0)
         autodetects your CPU count and uses that.
-    :param path blacklist_file: Path to a blacklist file, this file contains a
-        separate regex exclude on each newline.
-    :param path whitelist_file: Path to a whitelist file, this file contains a
-         separate regex on each newline.
+    :param path blacklist_file: DEPRECATED: soon to be replaced by the new
```

Comment:
I think you can remove the deprecated args here, we don't treat `test_processor` as a Stable API. See https://stestr.readthedocs.io/en/latest/api.html#internal-apis

Reply:
Changing "deprecation note" to simple mention of new args...